src/stateMachines/deactivation.ts

```diff
@@ -67,7 +67,10 @@ export class Deactivation {
             src: this.invokeUnprovision.bind(this),
             id: 'send-unprovision-message',
             onDone: 'REMOVE_DEVICE_FROM_SECRET_PROVIDER',
-            onError: 'ERROR'
+            onError: {
+              actions: assign({ errorMessage: (context, event) => 'Incorrect user name or password' }),
```

Comment:
Seems like we are always assuming username or password incorrect error. What if, its an error other than user name and password?

Reply:
updated to print parsed message payload